sme_finance_application_schema/finance_need_v1

```diff
@@ -30,6 +30,55 @@
                 "multiple"
             ]
         },
+        "type_of_mortgage": {
+            "title":"Mortgage type requested",
+            "enum":[ "owner_occupied",
+                "commercial_buy_to_let",
+                "residential_buy_to_let",
+                "refinance"
+            ]
+        },
+        "type_of_property": {
+            "title":"Property type requested",
+            "enum":["undeveloped_land_no_planning_permission",
+                "undeveloped_land_with_planning_permission",
+                "part_finished_development",
+                "buy_to_let",
+                "hmo",
+                "freehold_flats",
+                "holiday_homes",
+                "semi_commercial_unit",
+                "office",
+                "retail_unit",
+                "light_industrial",
+                "heavy_industrial",
+                "agricultural",
+                "warehousing"
+            ]
+        },
+        "experience_in_development": {
+            "enum":["None",
+                "refurbishement_and_sale",
+                "refurbishement_and_let",
+                "ground_up_development_single_unit",
+                "ground_up_development_multiple_units"
+                ]
+        },
+        "permission_for_development": {
+            "enum":["None",
+                "permitted_development",
+                "pre_application",
+                "outline_planning",
+                "detailed_planning"
+                ]
+        },
+        "equity_in_site": {
+            "enum":["None",
+                "between_50k_and_250k",
+                "more_than_250k",
+                "owned_outright"
```

Comment:
this mixes a proportion ("owned_outright") with values ("between_50k_and_250k") - what's my answer if it's worth £200k and i own it outright?